extensions/spark/kyuubi-spark-lineage/src/main/scala/org/apache/kyuubi/plugin/lineage/helper/SparkListenerHelper.scala

```diff
@@ -38,4 +39,6 @@ object SparkListenerHelper {
   def isSparkVersionEqualTo(targetVersionString: String): Boolean = {
     SemanticVersion(SPARK_VERSION).isVersionEqualTo(targetVersionString)
   }
+
+  def currentUser: String = UserGroupInformation.getCurrentUser.getShortUserName
```

Comment:
The current user from UGI may does not reflect the session user. We shall use session property ""kyuubi.session.user" as preferred one in the followups.

Reply:
> The current user from UGI may does not reflect the session user. We shall use session property ""kyuubi.session.user" as preferred one in the followups.

Thanks, I didn't find a suitable way to get the session user before. But, I think it is more appropriate to be set to `remoteUser`.